### DISCUSSION TITLE
0.66 keep getting removed by windows defender

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#430] [General]
0.66 keeps getting deleted by windows defender 
![image](https://github.com/seerge/g-helper/assets/132545384/c0d56b88-a25c-45bb-8a3e-cdef93dcd435)
any idea why?

**Comment by seerge** [comment]
@thduc87 hello. False positives can happen. You can always download a previous version or just recompile app by yourself. Or try this buld (just recompilled)  
[GHelper.zip](https://github.com/seerge/g-helper/files/11531468/GHelper.zip)

**Reply by thduc87** [reply]
thanks man, really appreciated
